### ISSUE TITLE
library "libssh2.so" not found

### ISSUE THREAD

**Issue by yochananmarqos** [NeoTerrm/NeoTerm#4]
I installed v1.2.1 via F-Droid several days ago and was able to run `apt update && apt upgrade`. I updated to v1.2.2 and now I receive this error:

```
$ apt update && apt upgrade
CANNOT LINK EXECUTABLE "/data/data/io.neoterm/files/usr/lib/apt/methods/https": library "libssh2.so" not found
Reading package lists... Done
E: Method https has died unexpectedly!
E: Sub-process https received signal 6.
E: Method /data/data/io.neoterm/files/usr/lib/apt/methods/https did not start correctly
E: Failed to fetch https://mirrors.geekpie.org/neoterm/dists/stable/InRelease
E: Some index files failed to download. They have been ignored, or old ones used instead.
```

Nexus 5X
Android 7.1.2

**Comment by imkiva** [comment]
This error has been solved.
Now you can change the source to `neoterm.studio` in the `Package Settings`(menu -> Package Settings), then do `apt upgrade`.

_Sent from my Meizu M5 Note using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by imkiva** [comment]
If there's no `neoterm.studio` in Package Settings, you can also use the `Debug source`.

_Sent from my Meizu M5 Note using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by licaon-kter** [comment]
Seeing this for "profanity", it installs ok but you can't run it, yes with `neoterm.studio` repo.

```
$ profanity --help
CANNOT LINK EXECUTABLE: library "libssh2.so" not found
page record for 0xb6e8e00c was not found (block_size=64)
```

**Comment by imkiva** [comment]
@licaon-kter So, have you installed libssh2 successfully? Checking that by running `apt list | grep libssh2`

**Comment by licaon-kter** [comment]
You are right, that fixes it, but shouldn't it be installed as a dependency if needed it in the first place?

**Comment by imkiva** [comment]
@licaon-kter yes, now it will be installed automatically.

**Comment by licaon-kter** [comment]
Still seeing this in 1.2.6, please reopen

clean app data
apt update
apt install git
git init
git clone <somerepo> -> error "libssh2.so"

**Comment by imkiva** [comment]
Sorry for that, I am working on this. Updates will be available tonight.

**Comment by imkiva** [comment]
Fixed in v2.0.0, check it out [here](https://github.com/NeoTerm/NeoTerm/releases/tag/v2.0.0)
